src/core/theme.cpp

```diff
@@ -86,6 +86,7 @@ Theme::Metadata Theme::readMetadata(const Palette &p_obj)
     data.m_revision = metaObj[QStringLiteral("revision")].toInt();
     data.m_editorHighlightTheme = metaObj[QStringLiteral("editor-highlight-theme")].toString();
     data.m_markdownEditorHighlightTheme = metaObj[QStringLiteral("markdown-editor-highlight-theme")].toString();
+    data.m_IconMonochrome = metaObj[QStringLiteral("icon-monochrome")].toBool();
```

Comment:
Add a default value `true` here in `toBool()` to be compatible with existing themes.